src/graphorm/index.ts

```diff
@@ -390,6 +391,27 @@ const obj = new GraphORM({
           parentColumn: 'parentId',
         },
       },
+      userState: {
```

Comment:
I think you can actually use `transform: nullIfNotLoggedIn,` here just to avoid the manual user exist and extra safety on ensuring it's your user?

Reply:
I can't since I also want to return defaults if there is not record. 
```js
if (!value) {
    return ctx.con.getRepository(UserComment).create();
 }
```

If `nullIfNotLoggedIn` would accept function then I would be able to do it.